src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorHelpers.cs

```diff
@@ -0,0 +1,624 @@
+						//{
+						//	break;
+						//}
+
+						// We want the checkered pattern to alternate both vertically and horizontally.
+						// In order to achieve that, we'll toggle visibility of the current pixel on or off
+						// depending on both its x- and its y-position.  If x == CheckerSize, we'll turn visibility off,
+						// but then if y == CheckerSize, we'll turn it back on.
+						// The below is a shorthand for the above intent.
+						bool pixelShouldBeBlank = ((x / CheckerSize) + (y / CheckerSize)) % 2 == 0 ? true : false;
+
+						if (pixelShouldBeBlank)
+						{
+							bgraCheckeredPixelData.Add(0);
+							bgraCheckeredPixelData.Add(0);
+							bgraCheckeredPixelData.Add(0);
+							bgraCheckeredPixelData.Add(0);
+						}
+						else
+						{
+							bgraCheckeredPixelData.Add((byte)(checkerColor.B * checkerColor.A / 255));
+							bgraCheckeredPixelData.Add((byte)(checkerColor.G * checkerColor.A / 255));
+							bgraCheckeredPixelData.Add((byte)(checkerColor.R * checkerColor.A / 255));
+							bgraCheckeredPixelData.Add(checkerColor.A);
+						}
+					}
+				}
+			});
+
+			//if (asyncActionToAssign != null)
+			//{
+			//	asyncActionToAssign.Cancel();
+			//}
+
+			//asyncActionToAssign = ThreadPool.RunAsync(workItemHandler);
+			//asyncActionToAssign.Completed = new AsyncActionCompletedHandler(
+			//async (IAsyncAction asyncInfo, AsyncStatus asyncStatus) =>
+			//{
+			//	if (asyncStatus != AsyncStatus.Completed)
+			//	{
+			//		return;
+			//	}
+
+			//	asyncActionToAssign = null;
+
+				// Uno Doc: Assumed normal priority is acceptable
+				await dispatcherHelper.RunAsync(CoreDispatcherPriority.Normal, () =>
+				{
+					WriteableBitmap checkeredBackgroundBitmap = CreateBitmapFromPixelData(width, height, bgraCheckeredPixelData);
+					completedFunction?.Invoke(checkeredBackgroundBitmap);
+				});
+			//});
+		}
+
+		public static WriteableBitmap CreateBitmapFromPixelData(
+			int pixelWidth,
+			int pixelHeight,
+			List<byte> bgraPixelData)
+		{
+			// IBufferByteAccess isn't included in any WinMD file, because its sole method - Buffer() -
+			// allows direct pointer access, which isn't applicable to C#.  In C#, there's a separate ToStream()
+			// method that similarly allows read access.  As a result, we have no C++/WinMD projection for IBufferByteAccess;
+			// we need to flip to ABI in this circumstance to do what this method requires.
+			WriteableBitmap bitmap = new WriteableBitmap(pixelWidth, pixelHeight);
+
+			// Uno Doc:
+			// Since Uno uses C#, the method of converting to a bitmap was changed to what is below.
+			// WARNING: The .ToArray() copy is a performance hit. It would be better to just pass arrays to this method.
+			using (Stream stream = bitmap.PixelBuffer.AsStream())
+			{
+				_ = stream.WriteAsync(bgraPixelData.ToArray(), 0, bgraPixelData.Count);
+			}
+
+			// Uno Doc: The following code is not supported in C# and is removed.
+			//byte *pixelBuffer = nullptr;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)